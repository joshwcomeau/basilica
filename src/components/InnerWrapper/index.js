// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import './index.scss';


const InnerWrapper = ({ children, ...delegated }) => {
  const classes = classNames(['inner-wrapper', delegated.className]);

  return (
    <div {...delegated} className={classes}>
      <div className="inner-wrapper-contents">
        {children}
      </div>
    </div>
  );
};

InnerWrapper.propTypes = {
  children: PropTypes.node,
};

InnerWrapper.defaultProps = {
};

export default InnerWrapper;
