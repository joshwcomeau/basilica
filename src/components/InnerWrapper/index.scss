@import '../../variables';

.inner-wrapper {
  position: relative;
  display: flex;
  justify-content: center;


  .inner-wrapper-contents {
    position: relative;
    width: 100%;
    max-width: $max-width-inner;
    margin-left: 40px;
    margin-right: 40px;
  }

  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
