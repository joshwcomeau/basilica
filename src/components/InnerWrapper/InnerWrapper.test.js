/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import InnerWrapper from './index';

describe('InnerWrapper', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<InnerWrapper />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
