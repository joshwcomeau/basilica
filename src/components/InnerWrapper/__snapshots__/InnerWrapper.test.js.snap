exports[`InnerWrapper renders correctly 1`] = `
<div
  className="inner-wrapper">
  Your Component Here :)
</div>
`;
